### PR TITLE
Prakash

### PR DESCRIPTION
**Pull request by prakash014**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,11 @@
 
 </div>
 
+<div>
+
```
Comment (coolprobn) [inline index.html]: why was this added?

Comment (prakash014) [inline index.html]: ok that was mistake
